modules/dists/SparseBlockDist.chpl

```diff
@@ -804,3 +806,22 @@ proc dropDims(D: domain, dims...) {
   return DResult;
 }
 
+proc SparseBlockDom.dsiHasSingleLocalSubdomain() param return true;
+proc SparseBlockArr.dsiHasSingleLocalSubdomain() param return true;
+
+proc SparseBlockDom.dsiLocalSubdomain() {
+  // TODO -- could be replaced by a privatized myLocDom in BlockDom
+  // as it is with BlockArr
+  var myLocDom:unmanaged LocSparseBlockDom(rank, idxType, stridable, sparseLayoutType) = nil;
+  for (loc, locDom) in zip(dist.targetLocales, locDoms) {
```

Comment:
This loop is clearly non-optimal, maybe call out the O(1) vs O(n) element in the comment above?